bridge_adaptivity/api/tests/test_api.py

```diff
@@ -114,7 +114,7 @@ def setUpTestData(cls):
             "source_name": "",
             "stype": "",
             "collection": cls.collection.id,
-            "lti_consumer": None
+            "lti_content_sources": None
```

Comment:
This and the same following changes looks wrong. Please clarify why do we need them?